cmd/util/cmd/epochs/transactions/deploy_epoch_relative.cdc

```diff
@@ -0,0 +1,31 @@
+import FlowClusterQC from 0xQCADDRESS
+
+transaction(name: String, 
+            code: [UInt8],
+            currentEpochCounter: UInt64,
+            numViewsInEpoch: UInt64,
+            numViewsInStakingAuction: UInt64,
+            numViewsInDKGPhase: UInt64,
+            numCollectorClusters: UInt16,
+            FLOWsupplyIncreasePercentage: UFix64,
+            randomSource: String,
+            collectorClusters: [FlowClusterQC.Cluster]) {
+
+  prepare(signer: AuthAccount) {
+
+    let currentBlock = getCurrentBlock()
+
+    signer.contracts.add(name: name, 
+            code: code,
+            currentEpochCounter: currentEpochCounter,
+            numViewsInEpoch: numViewsInEpoch - currentBlock.view,
```

Comment:
Let's add context: 

The smart contract computes the final view of the epoch like `currentBlock.view+numViewsInEpoch`. Since the Epoch contract is deployed _after_ the spork while the network is running, we need to adjust the `numViewsInEpoch` value based on the current block when epochs is deployed